README.md

```diff
@@ -53,6 +54,8 @@ Back in your webstore checkout:
 
 and your `webstore` will start using the local component build.
 
+If new dependencies are added to the project, you will need to temporarily delete the line `"@scientist-softserv/webstore-component-library": "VERSION_HERE",` from the `package.json` file in order to run yarn and build the project with new dependencies.
```

Comment:
is this actually the case? I thought the only problem with having the library dependency was that it needed to be removed if someone is developing with the local library instead of the published one.

Reply:
i went to double check, but i think it makes more sense to try after the ajv dependencies PR gets merged since there are several other changes in my package-lock file that will need to be reverted to merge in the new changes

Reply:
ok- just checked.

I am not able to run yarn in order to install new dependencies with the line above present. 
<img width="879" alt="image" src="https://user-images.githubusercontent.com/73361970/213221523-adeae52b-8734-4b15-ab4f-b3246add7248.png">

when i remove that line, i can run yarn without issue

Reply:
ohhh. you're getting the 401 unauthorized. have you set up your github token like it says on [these steps](https://github.com/scientist-softserv/webstore#configure-token-to-pull-from-the-github-npm-repository)?

Reply:
also, the ajv code is merged now. 🙂

Reply:
to explain: the 401 is because the component library is a github npm package, not a "regular" npm package. therefore, it wants a github token present to be able to use it. so when you ask the app to install the package, it sees that there isn't a github token, and denies you access. once you have your token in `~/.npmrc` (not to be confused with the file of the same name in the repo), you should be able to install it.

Reply:
ok- done! updated the readme too